app/main/views/search.py

```diff
@@ -110,7 +110,19 @@ def supplier_search():
                 }
             },
             "sort": sort_queries,
-        }
+            }
+
+    elif keyword:
+        query = {
+            "query": {
+                "wildcard": {
```

Comment:
I'm not sure if `wildcard` is the best query type for this. It can be quite slow and it doesn't necessarily describe what we're trying to achieve with keyword search. Some alternatives are:
* [`match_phrase`](https://www.elastic.co/guide/en/elasticsearch/guide/1.x/_phrase_search.html) if we want to match given string exactly (ie all words in a given order as a single phrase)
* [`match`](https://www.elastic.co/guide/en/elasticsearch/guide/1.x/_full_text_search.html) if we want full text search that won't check for words being in the right order
* [`simple_query_string`](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html) if we want to allow users to write simple queries using a query DSL (which allows grouping, and, or, not etc)